### ISSUE TITLE
Transfers into liabilties show up negativ

### ISSUE THREAD

**Issue by hr87** [firefly-iii/firefly-iii#6733]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely me, not you](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

Transactions from asset accounts into liabilities show up negative in both. The should be negative in the asset account and positive in the liability. 

Currently, transactions like these are a withdrawal, and handles as such on both sides. They should be transfers.

### Expected behaviour

The transactions should be transfers, negative on one side, positive on the other.

### Debug information

Happens on the demo site

### Steps to reproduce

Create a transaction in the demo side from an asset account into a liability. It shows up with the same sign on both sides.

**Comment by JC5** [comment]
Yes, they will always show up negative. That's intentional and not a bug.
